tests/__init__.py

```diff
@@ -1,9 +1,29 @@
+import inspect
 from contextlib import asynccontextmanager
-
+import pytest
 from scrapy import Request
 from scrapy.http.response.html import HtmlResponse
 from scrapy.utils.test import get_crawler
 
+from scrapy_playwright.handler import windows_get_result
+
+
+def windows_pytest_mark_asyncio(pytest_mark_asyncio):
+    def wrapper(*args, **kwargs):
+        if args and inspect.iscoroutinefunction(args[0]):
+
+            async def method_proxy(*x):
+                await windows_get_result(args[0](*x))
+
+            return pytest_mark_asyncio(method_proxy)
+        return windows_pytest_mark_asyncio(pytest_mark_asyncio(*args, **kwargs))
```

Comment:
This looks like an infinite loop, probably only avoided because all test functions are coroutine functions.

Reply:
Thinking about `@pytest.mark.asyncio(scope="class")`